jobs/tests/test_views.py

```diff
@@ -0,0 +1,107 @@
+# encoding: utf-8
+
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+
+from jobs.models import Job, Company
+
+
+class JobCreateTests(TestCase):
+    def setUp(self):
+        Company.objects.create(
+            name="My Company",
+            website="http://mycompany.com/"
+        )
+
+    def test_job_add_new_company_clean(self):
+        """Tests adding a new job with a new company"""
+        response = self.client.post(reverse('jobs:job_new'), {
```

Comment:
It would save repetition if you set up a dictionary with the context data in the set up, and then use it again in each test, just replacing the data you're changing each time.
In setUp:
```
self.context = {
            'company_name': 'My Company',
            'website': 'http://company.com',
            'contact_email': 'jobs@company.com',
            'title': 'Job offer',
            'description': 'description',
            'city': u'Kraków',
            'country': 'PL',
            'overwrite': True,
}  
```
Then in this test:
```
context = self.context
context['company_name] = 'New Company'
context['website'] = 'http://newcompany.com'
response = self.client.post(reverse('jobs:job_new), context)
```